packages/xo-server/src/xo-mixins/api.js

```diff
@@ -255,7 +255,7 @@ export default class Api {
         .slice(2),
       userId,
       userName,
-      userIp: session.get('user_ip'),
+      userIp: session.get('user_ip', undefined),
```

Comment:
Why no using the existing `userId` variable?

Reply:
`userId` displayed in the `IP` column?

Reply:
My bad, I misunderstood.